index.js

```diff
@@ -71,7 +71,7 @@ if (NODE_ENV === 'development') {
   }))
   app.use(require('browserify-dev-middleware')({
     src: path.resolve(__dirname, 'mobile'),
-    transforms: [require('jadeify'), require('caching-coffeeify'), require('babelify')]
+    transforms: [require('jadeify'), require('caching-coffeeify')]
```

Comment:
Since we're booting off of `mobile` and `desktop` now is this stuff just overwritten?